package.json

```diff
@@ -122,9 +122,11 @@
     "scripts": {
         "build": "shx rm -rf dist && tsc -b && oclif manifest && oclif readme --multi",
         "build:tar": "oclif pack tarballs",
+        "format": "prettier --write .",
+        "format:check": "prettier --check .",
         "lint": "eslint . --ext .ts --config .eslintrc",
         "postpack": "shx rm -f oclif.manifest.json",
-        "posttest": "yarn lint",
+        "posttest": "yarn lint && yarn format:check",
```

Comment:
should this be `pretest` instead of post?

Reply:
mostly concerned that this will "fail" the build / task, and then require a full re-run... when this is the usually the fastest failure point